src/express/index.js

```diff
@@ -21,7 +21,7 @@ app.use(express.static(path.resolve(PROJECT_DIR, PUBLIC_DIR)));
 
 app.use(`/`, mainRouter);
 app.use((req, res) => res.status(HttpStatusCode.NOT_FOUND).render(`errors/404`));
-app.use((error, req, res, next) => res.status(HttpStatusCode.ERROR).render(`errors/500`));
+app.use((error, req, res) => res.status(HttpStatusCode.INTERNAL_SERVER_ERROR).render(`errors/500`));
```

Comment:
не надо удалять next

Define error-handling middleware functions in the same way as other middleware functions, except error-handling functions have four arguments instead of three: (err, req, res, next). For example:

```
app.use(function (err, req, res, next) {
  console.error(err.stack)
  res.status(500).send('Something broke!')
})
```